patches/minecraft/net/minecraft/block/BlockChorusFlower.java.patch

```diff
@@ -0,0 +1,204 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockChorusFlower.java
++++ ../src-work/minecraft/net/minecraft/block/BlockChorusFlower.java
+@@ -17,6 +17,9 @@
+ import net.minecraft.util.EnumFacing;
+ import net.minecraft.util.math.BlockPos;
+ import net.minecraft.world.World;
++import net.minecraftforge.common.MinecraftForge;
++import net.minecraftforge.event.world.BlockEvent;
++import net.minecraftforge.fml.common.eventhandler.Event;
```

Comment:
My apologies, it was late and I was frustrated and didn't see the "more info here" link.